crates/jstz_cli/src/utils.rs

```diff
@@ -9,8 +9,8 @@ pub fn from_file_or_id(data_or_file: String) -> String {
 }
 
 fn get_stdin() -> String {
-    let lines: Result<Vec<_>, _> = io::stdin().lines().collect();
-    lines.expect("Can't read from stdin").join("\n")
+    let lines: io::Result<Vec<_>> = io::stdin().lines().collect();
+    lines.expect("Failed to read lines from stdin").join("\n")
```

Comment:
Should we definitely be using expect here? It doesn't really matter in the CLI, but an error might be better.

Reply:
We can probably use a `Result` here. Will fix 👍

Reply:
Fixed, it's slightly more messy. But it works :)